charts/node-analyzer/README.md

```diff
@@ -99,7 +99,6 @@ To check the integrity and the origin of the charts,  append the `--verify` flag
 The following table lists the configurable parameters of the Sysdig Node Analyzer chart and their default values.
 
 | Parameter                                                            | Description                                                                                                                        | Default                                                                                                                                                               |
-|----------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
@AlbertoBarba - as discussed in Slack, I have doubts about this change. The other chart READMEs have this row and I believe its' a requirement of the GitHub Markdown parser to separate the column headers.